generators/app/templates/src/redux/auth/actions.ejs

```diff
@@ -52,10 +68,11 @@ export const actionCreators = {
   }
   <%_ }_%>
 };
-<%_ } _%>
-<%_ if(!features.loginandsignup && features.onboarding) { _%>
+<%_ } else if (features.onboarding) { _%>
```

Comment:
i think that if we use this if, we should change the code above, because if this value is, then it is for many actions defined above

Reply:
if you want we can check this later